.eslintrc

```diff
@@ -40,6 +51,79 @@
+        "app/javascript/packages/document-capture/components/document-capture.tsx",
+        "app/javascript/packages/document-capture/components/document-side-acuant-capture.jsx",
+        "app/javascript/packages/document-capture/components/documents-step.jsx",
+        "app/javascript/packages/document-capture/components/file-image.jsx",
+        "app/javascript/packages/document-capture/components/file-input.tsx",
+        "app/javascript/packages/document-capture/components/hybrid-doc-capture-warning.spec.tsx",
+        "app/javascript/packages/document-capture/components/hybrid-doc-capture-warning.tsx",
+        "app/javascript/packages/document-capture/components/in-person-call-to-action.spec.tsx",
+        "app/javascript/packages/document-capture/components/in-person-call-to-action.tsx",
+        "app/javascript/packages/document-capture/components/in-person-location-full-address-entry-post-office-search-step.spec.tsx",
+        "app/javascript/packages/document-capture/components/in-person-location-full-address-entry-post-office-search-step.tsx",
+        "app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.spec.tsx",
+        "app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx",
+        "app/javascript/packages/document-capture/components/in-person-outage-alert.spec.tsx",
+        "app/javascript/packages/document-capture/components/in-person-outage-alert.tsx",
+        "app/javascript/packages/document-capture/components/in-person-prepare-step.spec.tsx",
+        "app/javascript/packages/document-capture/components/in-person-prepare-step.tsx",
+        "app/javascript/packages/document-capture/components/in-person-switch-back-step.tsx",
+        "app/javascript/packages/document-capture/components/in-person-troubleshooting-options.tsx",
+        "app/javascript/packages/document-capture/components/review-issues-step.tsx",
+        "app/javascript/packages/document-capture/components/status-message.jsx",
+        "app/javascript/packages/document-capture/components/submission-complete.tsx",
+        "app/javascript/packages/document-capture/components/submission-interstitial.jsx",
+        "app/javascript/packages/document-capture/components/submission-status.jsx",
+        "app/javascript/packages/document-capture/components/submission.jsx",
+        "app/javascript/packages/document-capture/components/suspense-error-boundary.jsx",
+        "app/javascript/packages/document-capture/components/tip-list.tsx",
+        "app/javascript/packages/document-capture/components/unknown-error.tsx",
+        "app/javascript/packages/document-capture/components/warning.tsx"
+      ],
+      "rules": {
+        "react/prop-types": "off",
+        "react/display-name": "off",
+        "react/jsx-key": "off",
+        "react-hooks/exhaustive-deps": "off",
+        "react-hooks/rules-of-hooks": "off"
+      }
```

Comment:
Acknowledging it's more work, but would it be more effective to have overrides and file listing for individual rules, so that someone can fix one rule violation and not all rule violations for any particular file?

Reply:
Interesting idea. I don't think it's worth it since there are few (less than 5) violations in many/most files as far as I've seen. Also, I'd expect that most of the harder ones to fix will be around hooks (`useEffect`) so it would make sense to address `rules-of-hooks` and `exhaustive-deps` at the same time. The violations of other rules have (so far) been pretty trivial to address.

Reply:
Ok, that's fair 👍